src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import { FC, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const TabsPage: FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+  const [selectedTab, setSelectedTab] = useState(true);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const {
+              id,
+              title,
+            } = tab;
+
+            return (
+              <li
+                data-cy="Tab"
+                className={classNames({ 'is-active': tabId === id })}
+                key={id}
+              >
+                <Link
+                  to={`/tabs/${id}`}
+                  onClick={() => setSelectedTab(false)}
+                >
+                  {title}
+                </Link>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {selectedTab
+          ? 'Please select a tab'
+          : `Some text ${tabId?.slice(-1)}`}
```

Comment:
```suggestion
        {selectedTab
          ? 'Please select a tab'
          : selectedTab.content}
```
There is no need to hardcode the content of the tab. You can get it from the selected tab